src/components/pagination-nav/pagination-nav.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import settings from '../../globals/js/settings';
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import InitComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+import handles from '../../globals/js/mixins/handles';
+
+class PaginationNav extends mixin(createComponent, InitComponentBySearch, handles) {
+  /**
+   * Pagination Nav component
+   * @extends CreateComponent
+   * @extends InitComponentBySearch
+   * @extends Handles
+   * @param {HTMLElement} element The element working as a pagination nav.
+   */
+  constructor(element, options) {
+    super(element, options);
+    this.element.addEventListener('click', evt => this.handleClick(evt));
```

Comment:
Carbon vanilla codebase has lifecycle management - `release()` method of this class should remove the event listener, to avoid event listener leak. We use [`on()`](https://github.com/IBM/carbon-components/blob/v9.70.0/src/globals/js/misc/on.js#L8-L16) utility function along with [`handles` mix-in](https://github.com/IBM/carbon-components/blob/v9.70.0/src/globals/js/mixins/handles.js) for that lifecycle management. Could you please do the same?